src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -470,4 +470,70 @@ private function writeText(string $content, array $options = array())
             isset($options['raw_output']) ? !$options['raw_output'] : true
         );
     }
+
+    private function describeMethods(array $methods)
```

Comment:
The descriptor does not only deal with routing (DI, event dispatching), I would rename to `describeRouteMethods()`/`describeRoutePath()`